src/repo/dune

```diff
@@ -4,7 +4,7 @@
  (modules Learnocaml_index
           Learnocaml_exercise)
  (libraries ocplib-json-typed
-            xor
+            learnocaml_xor
```

Comment:
How is that change related to the PR?

Reply:
Digestif has itself a module named Xor, and dune complained about the presence of two modules with the same name.